addon/-private/system/model/model-information-registry.js

```diff
@@ -0,0 +1,226 @@
+import Ember from 'ember';
+import { assert, warn } from "ember-data/-private/debug";
+import EmptyObject from "ember-data/-private/system/empty-object";
+import {
+  typeForRelationshipMeta,
+  relationshipFromMeta
+} from "ember-data/-private/system/relationship-meta";
+
+const {Map, MapWithDefault, get} = Ember;
+
+/**
```

Comment:
Currently the API docs list all such docs, despite the class being marked as `@private` (see emberjs/website#2452). Can you change this to `/*` so it doesn't show up in the API docs?